src/components/20-molecules/datepicker/index.scss

```diff
@@ -26,17 +42,23 @@
 
   &__input-wrap {
     display: flex;
-    margin-bottom: 1px;
-    border: solid 1px $color-prim-gray-mercury;
     width: 197px;
   }
 
   &__input-button {
     background: $color-prim-white;
-    cursor: pointer;
-    border: 0;
+    cursor: pointer;border: 1px solid;
+    border-color: $color-prim-gray-mercury;
+    border-left: none;
     line-height: 0;
     padding: 1px 3px 2px;
+    margin-left: 0; /* suppress Safari 2px margin */
```

Comment:
AFAIK such comments are not wanted by the team. Comments about the user agent style sheets that gets overridden.

Reply:
I just checked with Luca: that is not the position of the team. Mere logic would suggest that one needs to know whether an otherwise unneeded CSS property is imposed just to fix a browser quirk or bug. That is the case here. If I don't comment, we have information loss - so I comment. Hope that makes sense to you.